torch_tvm/operators.cpp

```diff
@@ -14,12 +20,50 @@ std::unordered_map<Symbol, TVMOpFunctor>& getTVMOperatorMap() {
   return map;
 }
 
-RegisterTVMOperator::RegisterTVMOperator(
-    std::vector<std::pair<Symbol, TVMOpFunctor>> ops) {
-  for (const auto& pair : ops) {
-    auto sym = std::get<0>(pair);
-    auto op = std::get<1>(pair);
-    getTVMOperatorMap()[sym] = op;
+// These "wrapper" graphs are used to store the subgraphs
+// that will be compiled during execution.
+static std::list<Graph> wrapper_graphs;
+RegisterTVMOperator::RegisterTVMOperator(std::vector<TVMOpMap> ops) {
+  for (const auto &op : ops) {
+    getTVMOperatorMap()[op.sym] = op.fn;
+
+    if (op.name != "") {
+      auto torch_ops = getAllOperatorsFor(op.sym);
+
+      for (const auto &torch_op : torch_ops) {
+
+        auto schema = torch_op->schema();
+
+        wrapper_graphs.emplace_back();
+        auto &wrapper_graph = wrapper_graphs.back();
+        std::vector<Value *> torch_inputs;
+        for (const auto &inp : schema.arguments()) {
+          torch_inputs.emplace_back(wrapper_graph.addInput());
+        }
+        Node* node =
+            wrapper_graph.create(op.sym, torch_inputs, schema.returns().size());
+        wrapper_graph.appendNode(node);
+        wrapper_graph.registerOutput(node->output());
```

Comment:
this is a bit confusing. `wrapper_graph` create and append the node into its graph, then it does not ever get used when we creating the `SingletonSubgraph`?

Reply:
it's a lifetime management hack